submitit/snapshot.py

```diff
@@ -0,0 +1,38 @@
+import fnmatch
+import os
+import shutil
+from subprocess import check_output
+
+
+def run_cmd(str_args, **kwargs):
+    return check_output(str_args, shell=True, **kwargs).decode("utf-8").strip()
+
+
+class SnapshotManager:
+    def __init__(
+        self, snapshot_dir: str, with_submodules: bool = False, exclude=None,
```

Comment:
generally submit uses Path instead of string to represent folder/file names. Could we use that here to ?